.github/workflows/node.js.yml

```diff
@@ -0,0 +1,61 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+          cache: 'npm'
+          cache-dependency-path: 'frontend/package-lock.json'
+      - name: Install dependencies
+        run: npm install --global npm@${{ env.NPM_VERSION }}
+
+      - name: "Install dependencies: workspace"
+        working-directory: frontend
+        run: |
+          npm install
+          # Work-around until Hydrogen is updated: we trick hydrogen into thinking its installed in the package path.
+          # See: https://github.com/hydrogen-design-system/hydrogen.css/issues/72
+          mkdir -p admin/node_modules common/node_modules talentsearch/node_modules
+          (cd admin/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+          (cd common/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+          (cd talentsearch/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+
+      - name: "Run Unit Tests: common"
+        working-directory: frontend/common
+        run: |
+          # No need to build since we only use admin's hydrogen.css.
+          #npm run h2-build
+          npm install --force
```

Comment:
This was run in `frontend/` already, so no need to re-run. I think this actually runs an install as if not a monorepo, regenerating `frontend/common/package-lock.json` in the process, which shouldn't exist. So it's likely bad to run again

```suggestion
```